### PR TITLE
Translate block labels

### PR DESCRIPTION
**Pull request by core23**
- [x] German
- [x] English
- [x] French
- [ ] Hungarian

### CODE REVIEW

**Comment by core23** [comment]
Translation please :) @rande @Soullivaneuh @sagikazarmark

File: Resources/translations/SonataNewsBundle.de.xliff [hunk 1]
```diff
@@ -443,6 +443,26 @@
...
+                <source>form.label_mode</source>
+                <target>Modus</target>
+            </trans-unit>
+            <trans-unit id="form.label_mode_public">
+                <source>form.label_mode_public</source>
+                <target>Öffentlich</target>
+            </trans-unit>
+            <trans-unit id="form.label_mode_admin">
+                <source>form.label_mode_admin</source>
+                <target>Admin</target>
```
Comment (OskarStark) [inline Resources/translations/SonataNewsBundle.de.xliff]: What does ``Admin`` mean?

Maybe something like ``Nur für Administratoren`` (``Only for Admins``) ??

Comment (core23) [inline Resources/translations/SonataNewsBundle.de.xliff]: If you want to render a block to show admin links, you have to choose ``Admin``.

Comment (OskarStark) [inline Resources/translations/SonataNewsBundle.de.xliff]: ah ok, hmm ok, what do you think if we show a help text there where you describe this?

**Comment by OskarStark** [comment]
@core23 i updated the initial post, what do you think?

**Comment by rande** [comment]
form.label_number => Nombre
form.label_title => titre
form.label_mode => mode 
form.label_mode_public => mode public
form.label_mode_admin => mode administration

File: Resources/translations/SonataNewsBundle.de.xliff (hunk 1 above)
Comment (core23) [inline Resources/translations/SonataNewsBundle.de.xliff]: I changed the label to ``Administrativ`` which would discribes the ``Modus`` the best.

**Comment by OskarStark** [comment]
Thank you @core23

**Pull request merged**